filebeat/module/postgresql/log/_meta/fields.yml

```diff
@@ -12,7 +12,7 @@
     - name: thread_id
       type: long
       description: >
-          Processs id
+          Processes id
```

Comment:
I think this should be singular

Reply:
agree. Seems the script thought `sss` = missing `e` :-)